src/resources/PlatformResources.ts

```diff
@@ -60,6 +61,7 @@ const resourcesMap: Array<{key: string; resource: typeof Resource}> = [
     {key: 'saml', resource: Saml},
     {key: 'schemaService', resource: SchemaService},
     {key: 'search', resource: Search},
+    {key: 'searchInterfaces', resource: SearchInterfaces},
```

Comment:
Will need to add a unit test https://github.com/coveo/platform-client/blob/master/src/resources/tests/PlatformResources.spec.ts